assets/js/components/settings/SettingsCardKeyMetrics.js

```diff
@@ -89,13 +70,7 @@ export default function SettingsCardKeyMetrics() {
 					{ isUserInputCompleted && (
 						<Row>
 							<Cell size={ 12 }>
-								<UserInputPreview
-									goTo={ goTo }
-									noHeader
-									noFooter
-									settingsView
-									showIndividualCTAs
```

Comment:
In two instances where the `UserInputPreview` is used, this 4 prop means the same thing, only one is sufficient.